lib/team_stats.rb

```diff
@@ -5,6 +5,16 @@ def initialize(games, teams, game_teams)
     super(games, teams, game_teams)
   end
 
+  def team_info(team_id)
+    {
+      "team_id" => team_id,
+      "franchiseid" => @teams[team_id].franchise_id.to_i,
+      "teamname" => @teams[team_id].teamname,
+      "abbreviation" => @teams[team_id].abbreviations,
+      "link" => @teams[team_id].link
+      }
+  end
```

Comment:
needs "test" in test name to run in minitest,
rspec has the result looking like:
```
expected = {
      "team_id" => "18",
      "franchise_id" => "34",
      "team_name" => "Minnesota United FC",
      "abbreviation" => "MIN",
      "link" => "/api/v1/teams/18"
    }
```